src/debug/di/breakpoint.cpp

```diff
@@ -203,11 +205,11 @@ HRESULT CordbFunctionBreakpoint::Activate(BOOL fActivate)
         pEvent->BreakpointData.vmDomainFile = m_code->GetModule()->GetRuntimeDomainFile();
         pEvent->BreakpointData.encVersion = m_code->GetVersion();
 
-        BOOL fIsIL = m_code->IsIL();
+        BOOL codeIsIL = m_code->IsIL();
 
-        pEvent->BreakpointData.isIL = fIsIL ? true : false;
+        pEvent->BreakpointData.isIL = codeIsIL ? true : false;
```

Comment:
BreakpointData.isIL is ambiguously named, but looking at the ee side code it is best interpreted as "Is the offset an IL offset?". Assuming you did the renaming above, it should be set to m_offsetIsIL. In the native offset 0 breakpoint case you are adding m_code->IsIL() is TRUE, but this value needs to be FALSE.